models/repo.go

```diff
@@ -1177,6 +1178,30 @@ func prepareRepoCommit(e Engine, repo *Repository, tmpDir, repoPath string, opts
 		}
 	}
 
+	// Issue Labels
```

Comment:
This should be moved out to separate function as this is not related to initializing `git` structure and function call should be added to https://github.com/go-gitea/gitea/blob/cf8eb1ab9ce3f882b399601292f456023a6332dc/models/repo.go#L1408

Reply:
Done